client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -90,31 +190,155 @@ const EditProfilePage = () => {
+          />
+          <EditProfileInput
+            label="Current Title"
+            type="text"
+            name="currentPositionTitle"
+            value={formData.currentPositionTitle}
+            onChange={handleChange}
+          />
+          <label className="block font-bold mb-2 text-sm" htmlFor="availabilityForNetworking">
+            Available for Networking?
+            <div
+              className={`cursor-pointer h-3 inline-block ml-2 mt-2 w-3 ${
+                formData.availabilityForNetworking ? 'bg-yellow-500' : 'bg-blue-500'
+              }`}
+              role="button"
+              tabIndex={0}
+              onClick={() =>
+                setFormData((prevData) => ({
+                  ...prevData,
+                  availabilityForNetworking: !prevData.availabilityForNetworking,
+                }))
+              }
+            ></div>
+            <span className="ml-2 text-xs">
+              {formData.availabilityForNetworking ? 'Yes' : 'No'}
+            </span>
           </label>
-          <label className="block font-bold mb-2 text-sm" htmlFor="personalBio">
-            Personal Bio
-            <input
-              className="bg-gray-800 p-2 rounded text-white w-full"
-              id="personalBio"
-              name="personalBio"
-              type="text"
-              value={formData.personalBio}
-              onChange={handleChange}
-            />
+          <EditProfileInput
+            label="LinkedIn"
+            type="text"
+            name="linkedInProfile"
+            value={formData.linkedInProfile}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="GitHub"
+            type="text"
+            name="gitHubProfile"
+            value={formData.gitHubProfile}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Twitter"
+            type="text"
+            name="twitter"
+            value={formData.twitter}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Blog"
+            type="text"
+            name="blog"
+            value={formData.blog}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Skills"
+            type="text"
+            name="skills"
+            value={skillInput}
+            onChange={handleSkillChange}
+            onKeyDown={handleSkillKeyDown}
+            placeholder="Type a skill and press Enter"
+          />
+          <div className="flex flex-wrap gap-2 mb-2 mt-4">
+            {formData.skills.map((skill, index) => (
+              <div key={index} className="mb-2 mr-2 " onClick={() => handleSpecialization(skill)}>
+                <span
+                  // kept to avoid key warning
+                  key={index}
+                  className="bg-blue-500 incline-flex items-center px-4 py-1 rounded-full text-sm text-white"
```

Comment:
Here and line 326 just a little typo for inline-flex - i wish they had named it incline-flex